src/resources/workshops.yml

```diff
@@ -1,91 +1,46 @@
-      develop a Bot using it and show you that you have the skills to do this
-      too. You will also be introduced to the Cognitive Services APIs and see
-      how you can use them to transform your applications through a simple API
-      call.
-
-      About the Presenter: Lilian is a Technical Evangelist within the Developer
-      Experience Group (DX) at Microsoft. She is a Computer Science graduate,
-      tech enthusiast and IoT hobbyist. Lilian has previously worked as a
-      Software Engineer developing Mobile and Web applications. Her areas of
-      interest include Cognitive Services, Messenger Bots, Xamarin and Mobile
-      Development. See [http://liliankasem.com](http://liliankasem.com)
-
-    time: 2016-01-28 16:00:00
+    time: 2016-01-28 15:15:00
     duration: 30 # In minutes
     location: The Guildhall
-
-  - name: Using financial ratios to predict company bankruptcies by QuantumBlack
+  - name: Improbable Workshop
     description: |
-      QB has been mandated to help a national supervisory body rethink the way
-      they assess the financial health of companies they monitor. The objective
-      is to design a data driven approach that will help predict company
-      bankruptcies and subsequently enable identification of the best remedies.
 
-      Our workshop will cover how you can compete in our challenge to get ahead
-      of the game at Hack Cambridge Recurse!
-
-    time: 2016-01-28 16:45:00
+    time: 2016-01-28 15:50:00
     duration: 30 # In minutes
     location: The Guildhall
-
-  - name: How to get ahead with Unity by Studio Gobo
+  - name: Cambridge Consultants Workshop
     description: |
-      Ever wanted to make a game but never know where to start? Feel like you
-      know what you're doing, but feel like you could always learn some more?
-      Then swing by our workshop where a professional Game Developer will walk
-      through how to make a game in the Unity game engine, covering both the
-      basics, in addition to revealing tips and tricks that you may not know
-      about, based on years of experience in the industry.
 
-    time: 2016-01-28 17:30:00
+    time: 2016-01-28 16:25:00
+    duration: 30 # In minutes
+    location: The Guildhall
+  - name: Microsoft Cognitive Services Workshop
+    description: This workshop is aimed at supporting those interested in the Microsoft Challenge.  We will be providing a overview and demos supporting your innovative applications of Microsoft Cognitive Services http://aka.ms/cognitive. The challenge focus will around and the Vision API sets of cognitive services. AI, Data and pervasive technology which are driving the future. The team solutions will need to a built using Microsoft Cognitive Services. Cognitive Services are a free, easy-to-use, open-source SDK and services and even export models to run offline, starting with export to the CoreML format for iOS 11 we will also provide attendees with $100 of Azure for 3 months.
```

Comment:
It'd be nice to wrap this description (and use `|`, like the previous examples).